test/bin/manage_build_cache.sh

```diff
@@ -7,7 +7,8 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
 
-AWS_BUCKET_NAME="${AWS_BUCKET_NAME:-microshift-build-cache}"
+#AWS_BUCKET_NAME="${AWS_BUCKET_NAME:-microshift-build-cache}"
+AWS_BUCKET_NAME="microshift-pr-5548-cache-us-west-2"
```

Comment:
Leftover?

Reply:
yes, I'll remove this right now, this is a temp change to use a separate cache bucket
removed in 44ff05485084a73813cef9111454e7a5a97bf439 commit